src/validation.ts

```diff
@@ -1,13 +1,13 @@
 
-function isString(value) {
+function isString(value: any): boolean {
     return typeof value === 'string' || value instanceof String;
 }
 
-function isObject(value) {
+function isObject(value: any): boolean {
     return value && typeof value === 'object' && value instanceof Object;
 }
 
-function isMessageAttributeValid(messageAttribute) {
+function isMessageAttributeValid(messageAttribute: any): any {
```

Comment:
Technically, this function returns boolean. Although in practice reads "always true or throw Error". This needs refactoring but can be done when the basic TS conversion is complete.

Reply:
agreed!